### ISSUE TITLE
question about datasets

### ISSUE THREAD

**Issue by MASTERCHIEF343** [HKUST-Aerial-Robotics/MVDepthNet#19]
Hi, I'm working on my network.And i use rgbd and mvs datasets for training.But I find that mvs isn't work very well, the test accuracy is bad.But the training loss is very well and accuracy for rgbd test datasets is similliar with you(and I set depth plane as 16, because i see in your paper that 16 depth planes already has good results).Have you ever met that situation?

**Comment by WANG-KX** [comment]
Hi, the mvs dataset contains lots of occlusion and large view angle changes thus is very challenging. If you dig into the dataset, you will find many tree objects with terrible ground truth annotation. Since pixels that occluded can only be estimated by the smooth assumption or dataset priors, I suggest you use a deeper or wider network.
In my current opinion, DeMoN datset is not the best for motion stereo learning (because the occlusion and inaccurate camera poses), unless you are oriented for academic publication.

**Comment by MASTERCHIEF343** [comment]
Hi, thanks to your suggestion.Although I never use sun3d and scenes11 for training(I see other papers' writer use both of the four datasets), I'll try it and see the result.
